CHANGELOG.md

```diff
@@ -218,6 +218,9 @@ regression test suite and patches at https://github.com/seqan/seqan3/tree/master
    ([\#2556](https://github.com/seqan/seqan3/pull/2556))
 * Deprecated `seqan3::views::persist`. There is no replacement, use lvalues instead of rvalues
   ([\#2553](https://github.com/seqan/seqan3/pull/2553)).
+* We deprecated all upper case `seqan3::translation_frames::*FRAME*` and they will be removed in 3.1.0. Use
+  lower case frames `seqan3::translation_frames::*frame*` instead.
+  ([\#2565](https://github.com/seqan/seqan3/pull/2565))
```

Comment:
```suggestion
* Renamed `seqan3::translation_frames::*FRAME*` ([\#2565](https://github.com/seqan/seqan3/pull/2565)):
```

And then there needs to be a list (we did not just do upper to lower case, but also changed the names itself).
See the changelog entry for `([\#2459](https://github.com/seqan/seqan3/pull/2459))` for an example